core/DataAccess/LogAggregator.php

```diff
@@ -1134,13 +1138,13 @@ public static function getSelectsFromRangedColumn($column, $ranges, $table, $sel
 
                 $selectAs = "$selectColumnPrefix$lowerBound-$upperBound";
 
-                $selects[] = "sum(case when $table.$column between $lowerBound and $upperBound $extraCondition" .
+                $selects[] = "sum(case when $column between $lowerBound and $upperBound $extraCondition" .
```

Comment:
do you remember why $table was removed?

Reply:
It's handled above, I've made the code clearer.